spec/requests/posts_spec.rb

```diff
@@ -2,7 +2,7 @@
 
 RSpec.describe 'Posts', type: :request do
   describe 'posts index action' do
-    before(:example) { get user_posts_path(user_id: User.first.id) }
+    before { get '/users/1/posts' }
 
     it 'return a successful response' do
       expect(response).to have_http_status(200)
```

Comment:
- [ ] Some of the request and model specs are failing. I have attached the screenshot below of the test failure. Please fix or update the test according to the application.

![Screenshot from 2022-02-12 02-22-42](https://user-images.githubusercontent.com/22434660/153668785-a26ce1db-589a-4b44-b9d3-77f2fb7c03db.png)